src/sql/azdata.proposed.d.ts

```diff
@@ -1255,19 +1259,30 @@ declare module 'azdata' {
 			 */
 			type: DesignerEditType;
 			/**
-			 * the property that was edited
+			 * the path of the edit target.
 			 */
-			property: DesignerEditIdentifier;
+			path: DesignerEditPath;
 			/**
 			 * the new value
 			 */
 			value?: any;
 		}
 
 		/**
-		 * The identifier of a property. The value is string typed if the property belongs to the root object, otherwise the type of the value is an object.
+		 * The path of the edit target.
+		 * Table designer support nested tables, Below are the 3 scenarios and their expected path format.
+		 * 1. 'Add' scenario
+		 *    a. Level1Property. example: add a column to the columns property.
+		 *    b. Level1Property/index1/Level2Property. example: add a column mapping to a foreign key.
```

Comment:
As mentioned in the other PR - are there actual names in here (i.e. is `index1` a name?) 

If so then how are you handling names that contain /'s in them?

I looked through the code and I'm not sure I understand how the path is actually used since the code seems to use indexes/property names/field names directly instead of names but the examples here don't seem to have that. 

(even still, the property/field names can also contain / in them, right? Edge case for sure but something to at least document and preferably prevent a provider from declaring in the first place...)

Reply:
I've updated the comment here and add a comment about not using `/` in the property name.